### ISSUE TITLE
Newbie in Ruby on Rails, please help me get maily herald running.

### ISSUE THREAD

**Issue by dest4131** [Sology/maily_herald#14]
Dear maily_herard team,
first, a big thank you for publishing this interesting piece of software. There aren't much satisfying free alternatives in the world of email marketing. For example, PHPList's user experience is too crappy in my opinion.

I'm having a hard time running your software. I have the feeling that the documentation you have written is intended for experienced Ruby on Rails developers, and I am not (even if I have experience in administration otherwise). Could you please elaborate on the precise steps to do from a stock Debian distro for example? For example, apt-get install this and that... setup database with $ .. and $ ...

After hours of struggle, I could successfully run the sample_app, but the emails in the lists are locked (there are a few of them prefilled, people I don't know and I can't remove - I am afraid I could spam them). I am told to edit a file somewhere, but it's too much now and I want to give up.

Something else is not clear to me. I followed various tutorials and have the impression to have setup two database systems: postgresql AND redis. Are both dbs useful?

In a nutshell: please improve documentation for RoR newbies.

**Comment by ljachymczyk** [comment]
Hi @dest4131,

Thank you for giving maily_herald a try. We wanted to make it as easy to use as possible. However, you still need to have some basic knowledge to get it running. The maily_testapp you tried is a sample piece of code demonstrating Maily's capabilities by generating some fake email addresses.

I strongly encourage you to make MailyHerald your default tool for email marketing. If you need help with that, please contact us via www.sology.eu
